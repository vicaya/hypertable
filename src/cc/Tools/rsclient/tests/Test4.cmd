UPDATE 'test/Test4' "Test4-data.txt";
CREATE SCANNER ON 'test/Test4'[..??];
DESTROY SCANNER;
CREATE SCANNER ON 'test/Test4'[..??] RETURN_DELETES;
DESTROY SCANNER;
quit;
