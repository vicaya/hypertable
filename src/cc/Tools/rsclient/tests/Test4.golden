
Welcome to the rsclient command interpreter.
For information about Hypertable, visit http://www.hypertable.org/

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

UPDATE 'test/Test4' "Test4-data.txt";
success
CREATE SCANNER ON 'test/Test4'[..??];
1181153477000234 photoalbum plum:http://terraserver-usa.com/image.aspx?s=14&lon=-75.684776&lat=39.620362&w=2
1181153477000269 pseudomitotic broccoli:http://maps.yahoo.com/maps_result?lat=40.787484&lon=-76.538358&mag=4
1181153477000189 somatology apple:http://www.imdb.com/name/nm0071818/
1181153477000314 twentymo yam:http://www.sld.gov.tw/english/index.htm
DESTROY SCANNER;
CREATE SCANNER ON 'test/Test4'[..??] RETURN_DELETES;
1181153477000081 immotile plum DELETE
1181153477000018 overbuild broccoli:http://www.deltacountymi.org DELETE
1181153477000219 photoalbum plum DELETE
1181153477000234 photoalbum plum:http://terraserver-usa.com/image.aspx?s=14&lon=-75.684776&lat=39.620362&w=2
1181153477000100 postbrachium DELETE
1181153477000100 postbrachium DELETE
1181153477000100 postbrachium DELETE
1181153477000100 postbrachium DELETE
1181153477000263 pseudomitotic DELETE
1181153477000263 pseudomitotic DELETE
1181153477000263 pseudomitotic DELETE
1181153477000263 pseudomitotic DELETE
1181153477000269 pseudomitotic broccoli:http://maps.yahoo.com/maps_result?lat=40.787484&lon=-76.538358&mag=4
1181153477000168 somatology apple:http://www.imdb.com/name/nm0071818/ DELETE
1181153477000189 somatology apple:http://www.imdb.com/name/nm0071818/
1181153477000305 twentymo DELETE
1181153477000305 twentymo DELETE
1181153477000305 twentymo DELETE
1181153477000305 twentymo DELETE
1181153477000294 twentymo cassis DELETE
1181153477000281 twentymo cassis:http://terraserver-usa.com/image.aspx?s=14&lon=-75.684776&lat=39.620362&w=2 DELETE
1181153477000314 twentymo yam:http://www.sld.gov.tw/english/index.htm
DESTROY SCANNER;
